### PR TITLE
Fix for Firefox

### PR DESCRIPTION
**Pull request by jnishiyama**
Resolves issue #2

### CODE REVIEW

File: file-download.js [hunk 1]
```diff
@@ -3,14 +3,19 @@ var React = require('react');
...
+        // These URLs will no longer resolve as the data backing 
+        // the URL has been freed."
+        window.navigator.msSaveBlob(blob, filename);
+    }
+    else {
         var csvURL = window.URL.createObjectURL(blob);
         var tempLink = document.createElement('a');
         tempLink.href = csvURL;
         tempLink.setAttribute('download', filename);
-        tempLink.setAttribute('target', '_blank');
```
Comment (kennethjiang) [inline file-download.js]: Did this line break your solution? If not can you add it back? @josiahsprague added this line in PR #1 for a good reason.

**Comment by josiahsprague** [comment]
Probably just needs a rebase on master @jnishiyama :)

**Comment by jnishiyama** [comment]
target _blank is back in, didn't even realize I removed it, my apologies!

**Comment by tymochko** [comment]
Thanks for this fix. Although, Master branch on github is updated but for some reason it doesn't update on my machine.
I had it installed previously and wanted to update changes, so
I tried **npm install react-file-download --save** again or **npm update** and it didn't work.

As a workaround I succeeded by
**npm install https://github.com/kennethjiang/react-file-download/tarball/master**

**Comment by josiahsprague** [comment]
@tymochko You can use https://www.npmjs.com/package/npm-check-updates to update your dependencies, or to do it manually remove `react-file-download` from your package.json, run `rm -rf node_modules/react-file-download`, then `npm install react-file-download --save`. This applies any time you need to update a dependency, not just to `react-file-download`.

**Comment by josiahsprague** [comment]
`npm install react-file-download@0.3.1 --save` should also work.

**Comment by tymochko** [comment]
@josiahsprague, sorry, non of these did not work for me.

Besides I tried `rm -rf node_modules/react-file-download` - and it was not right, I had to write `npm rm -rf node_modules/react-file-download`

**Comment by kennethjiang** [comment]
This fix is not in 0.3.1. Just published 0.3.2. please upgrade.

**Pull request merged**